src/services/exercises.ts

```diff
@@ -51,8 +52,29 @@ const deleteExercise = async (id: string) => {
   return deleteRequest(`exercises/${id}`);
 };
 
+
+const edit = async (id:string,title: string, text: string, complexity: string) => {
+  return jsonPatch(`exercises/${id}`, {
+    title,
+    text,
+    complexity: complexity.toLowerCase(),
+  })
+    .then((response) => {
+      console.log(response.data);
+    })
+    .catch((error) => {
+      return Promise.reject(error);
+    });
```

Comment:
```suggestion
  return jsonPatch(`exercises/${id}`, {
    title,
    text,
    complexity: complexity.toLowerCase(),
  });
```

Reply:
If you dont do anything with the data itself, just return it normally

Reply:
Done